rails/extensions/nested_attributes_ext.rb

```diff
@@ -127,7 +127,7 @@ def has_many_type(association)
       reflection = association.reflection
       if reflection.try(:has_and_belongs_to_many?) ||
          (reflection.try(:parent_reflection) &&
-          reflection.parent_reflection.last.try(:macro) == :has_and_belongs_to_many)
+          reflection.parent_reflection.try(:macro) == :has_and_belongs_to_many)
```

Comment:
this is an actual change...?

Reply:
Yeah, parent_reflection before returned results as part of an array but no more.